lib/server/services/log/LogService.js

```diff
@@ -85,6 +85,40 @@ class LogService {
         return { count, logIds };
     }
 
+    /**
+     * Retrieves a list of replies to a log based off the given logId
+     * @param {number} logId The id of the log of which the child logs are retrieved
+     * @return {Promise<{count: number, childLogs: Log[]}>} A promise that resolves with an object containing the total count of all child logs,
+     * and an array of the child logs for the given parent log
+     */
+    async getChildLogsByParentId(logId) {
+        const { count, childLogIds } = await this._getChildLogIds(logId);
```

Comment:
I am not sure why you fetch id first here? I think that you can directly fetch the logs no?

Reply:
I do that so I can use _getAllByIds, which has some extra functionalities. Would it be better to just implement those functionalities directly and fetch the logs instead of id's?